src/ray/raylet/worker_pool.h

```diff
@@ -597,7 +597,7 @@ class WorkerPool : public WorkerPoolInterface, public IOWorkerPoolInterface {
   /// Try to find a task that is associated with the given worker process from the given
   /// queue. If found, invoke its PopWorkerCallback.
   /// \param workers_to_tasks The queue of tasks which waiting for workers.
-  /// \param proc The process which the worker belongs to.
+  /// \param startup_token The counter representing the worker.
```

Comment:
nit: The startup token representing the worker

Reply:
ok